src/components/Countdown/Countdown.spec.js

```diff
@@ -1,32 +1,9 @@
 import { screen, render } from '@testing-library/react'
-import userEvent from '@testing-library/user-event'
 import Countdown from './Countdown'
 
-jest.useFakeTimers()
-
 describe('Countdown', () => {
-  it('renders the countdown display and one button', () => {
-    render(<Countdown />)
+  it('renders the countdown display with given minutes and seconds', () => {
+    render(<Countdown minutes={25} seconds={0} />)
     expect(screen.getByText('25:00')).toBeInTheDocument()
-    expect(screen.getAllByRole('button')).toHaveLength(1)
-  })
-
-  it('calls setTimeout by clicking the button', () => {
-    render(<Countdown />)
-    expect(setTimeout).toHaveBeenCalledTimes(0)
-    userEvent.click(screen.getByRole('button'))
-    expect(setTimeout).toHaveBeenCalledTimes(1)
-    expect(setTimeout).toHaveBeenLastCalledWith(expect.any(Function), 1000)
-  })
-
-  it('does not call setTimeout function by clicking the button a second time, but does on a third time', () => {
-    render(<Countdown />)
-    const button = screen.getByRole('button')
-    userEvent.click(button)
-    userEvent.click(button)
-    expect(setTimeout).toBeCalled()
-    expect(setTimeout).toHaveBeenCalledTimes(1)
-    userEvent.click(button)
-    expect(setTimeout).toHaveBeenCalledTimes(2)
```

Comment:
Your tests' premises make a lot of sense. Why did you get rid of them?

Reply:
Unfortunately I had to remove them, since the logic is not part of the component anymore. For a comprehensive integration test, I will use Cypress later on.